app/src/main/java/net/gsantner/markor/format/TextConverter.java

```diff
@@ -43,7 +44,7 @@
     protected static final String TOKEN_POST_TODAY_DATE = "{{ post.date_today }}";
 
     protected static final String HTML_DOCTYPE = "<!DOCTYPE html>";
-    protected static final String HTML001_HEAD_WITH_BASESTYLE = "<html><head>" + CSS_S + "html,body{padding:4px 8px 4px 8px;font-family:'" + TOKEN_FONT + "';}h1,h2,h3,h4,h5,h6{font-family:'sans-serif-condensed';}a{color: " + TOKEN_LINK_COLOR + ";text-decoration:underline;}img{height:auto;width:325px;margin:auto;}" + CSS_E;
+    protected static final String HTML001_HEAD_WITH_BASESTYLE = "<html lang='" + Locale.getDefault().getLanguage() + "'><head>" + CSS_S + "html,body{padding:4px 8px 4px 8px;font-family:'" + TOKEN_FONT + "';}h1,h2,h3,h4,h5,h6{font-family:'sans-serif-condensed';}a{color: " + TOKEN_LINK_COLOR + ";text-decoration:underline;}img{height:auto;width:325px;margin:auto;}" + CSS_E;
```

Comment:
does not respect the app's set language. it's static and thus will use the system language. you need to set the value in the convert method, you can't do it with static var properly

Reply:
Doesn't `Locale.getDefault().getLanguage()` return the app language?

Reply:
* start app, use system settings to change language, see wrong language
* start app, change language inside app's settings, see wrong language

Reply:
Should be working now.